ReduxMovieDB/Controllers/MovieListViewController.swift

```diff
@@ -91,6 +93,10 @@ class MovieListViewController: UIViewController {
         super.viewWillDisappear(animated)
         mainStore.unsubscribe(self)
     }
+    
+    @IBAction func onToggleFavorites(_ sender: Any) {
```

Comment:
This would be better done with RxSwift in didSet of favoritesToggleItem (see above for examples)

Reply:
Also, I think the button should be a full star when it's in favorites mode (★)

Reply:
Got it 👌